tests/jballerina-semtype-port-test/src/test/java/io/ballerina/semtype/port/test/SemTypeTest.java

```diff
@@ -151,49 +161,74 @@ public void listAllBalFiles(File file, List<File> balFiles) {
         if (file.isFile()) {
             return;
         }
-        for (File f : file.listFiles()) {
+        for (File f : Objects.requireNonNull(file.listFiles())) {
```

Comment:
Is there a specific reason for calling `Objects.requireNonNull` here?. I think if `file.listFiles()` is `null` you are anyway going to get the null pointer exception here even without `requireNonNull` since loop is going to call `.iterator()`

Reply:
Removed. Added that change as a warning was given by IDE for this line.